.github/workflows/publish-go-nightly-task.yml

```diff
@@ -144,9 +157,6 @@ jobs:
           tar -czvf "$PACKAGE_FILENAME" \
           -C "${{ env.PROJECT_NAME }}_osx_${{ matrix.artifact.name }}/" "${{ env.PROJECT_NAME }}" \
           -C ../../ LICENSE.txt
-          CHECKSUM_LINE="$(shasum -a 256 $PACKAGE_FILENAME)"
```

Comment:
You forgot to remove checksum in the name of this step:
https://github.com/arduino/arduino-cli/pull/1883/files#diff-b01c8ff87205e83918583aff246152ef27d286cccaa14409dddc17bd37104518R143